web/src/components/profile/ProfileErrorAlert.tsx

```diff
@@ -15,6 +15,7 @@ export const ProfileErrorAlert = (props: Props): ReactElement | null => {
   const displayProfileErrorAlert = (): boolean => props.fieldErrors.length > 0;
 
   const getLabel = (field: string): string => {
+    // This is tech debt; if you do need to disable typescript please include a comment justifying why.
```

Comment:
I might modify this slightly in all instances. In some cases, I'm not sure that I would classify this as technical debt.

In this specific case, we're running into an issue where we cannot easily predict a type. We're not doing something "wrong", but it doesn't nicely conform to TypeScript's rules. I would say something similar for usages related to migrations. There's a pretty good reason for disabling this rule in those locations.

```suggestion
    // If you do need to disable typescript please include a comment justifying why.
```